.github/workflows/test.yml

```diff
@@ -69,6 +69,9 @@ jobs:
     - name: Run11
       run: |
         clox --calendar=year --date-system=jalAli --country=De
+    - name: Run12
```

Comment:
Add this run as `Run 14` at the end of the list or adjust the run numbers accordingly.